src/components/filopplaster/Filopplaster.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useCallback, useState } from 'react';
+import { useDropzone } from 'react-dropzone';
+import useSøknadContext from '../../context/SøknadContext';
+import { injectIntl, IntlShape } from 'react-intl';
+import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import opplasting from '../../assets/opplasting.svg';
+import OpplastedeFiler from './OpplastedeFiler';
+import { formaterFilstørrelse } from './utils';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import Modal from 'nav-frontend-modal';
+
+interface Props {
+  intl: IntlShape;
+  tittel: string;
+  beskrivelsesListe?: string[];
+  tillatteFiltyper?: string[];
+  maxFilstørrelse?: number;
+}
+
+const Filopplaster: React.FC<Props> = ({
+  intl,
+  tittel,
+  beskrivelsesListe,
+  tillatteFiltyper,
+  maxFilstørrelse,
+}) => {
+  const { søknad, settSøknad } = useSøknadContext();
+  const [feilmeldinger, settFeilmeldinger] = useState<string[]>([]);
+  const [åpenModal, settÅpenModal] = useState<boolean>(false);
+
+  const lukkModal = () => {
+    settÅpenModal(false);
+  };
+
+  const onDrop = useCallback((filer) => {
+    const data = søknad.vedlegg;
+    const liste: string[] = [];
+
+    filer.forEach((fil: File) => {
+      const filKey = fil.name + fil.size;
+
+      if (maxFilstørrelse && fil.size > maxFilstørrelse) {
+        const maks = formaterFilstørrelse(maxFilstørrelse);
+        liste.push(fil.name + ' er for stor. Den må være under ' + maks + '.');
```

Comment:
Feilmeldinger bør vel også kunne vises på engelsk. Kunne man brukt intl for å lage en språkspesifikk melding her? 
Jeg har f.eks. laget en tekst for Forside.tsx der man erstatter [0] i teksten med navnet til søker. Her kunne man vel kanskje gjort det samme ved å ha [0] for fil navn og [1] for maks variabelen ^^ ✨

Reply:
Ja, fikset det her b39c2f8 :)